chirps/asset/models.py

```diff
@@ -39,7 +39,9 @@ class BaseAsset(PolymorphicModel):
 
     def scan_is_active(self) -> bool:
         """Return True if the asset is currently being scanned."""
-        return self.scan_run_assets.filter(~Q(scan__status='Complete')).exists()
+        return self.scan_run_assets.filter(
+            ~Q(scan__status='Complete') & ~Q(scan__status='Failed') & ~Q(scan__status='Canceled')
+        ).exists()
```

Comment:
Unrelated, but having scans in terminal states other than `Complete` was disabling the APIEndpointAsset edit button, and that was annoying